blog/2024-09-07-shadow-bucket-thumbnails/index.md

```diff
@@ -0,0 +1,237 @@
+---
+slug: shadow-bucket-thumbnails
+title: Using Shadow Buckets for Fun and Thumbnails
+description: >
+  Exploring Tigris' shadow bucket feature for fun and thumbnails. In the blog
+  post the feature will be used in an less conventional fashion. An additional
+  elixir service will generate thumbnails, where source files as well as
+  generated thumbnails are stored on Tigris, while making use of the included
+  CDN features of Tigris.
+image: ./a-pair-of-scissors-cutting-a-photograph.jpg
+keywords: [object storage, blob storage, s3, thumbnail, images, cloudfront, cdn]
+authors: [bmi]
+tags: [object storage, blob storage, s3, thumbnail, images, cloudfront, cdn]
+---
+
+# Using Shadow Buckets for Fun and Thumbnails
+
+Generating thumbnails of images has historically been a problem that needed to
```

Comment:
suggest adding "let's make it easier with a nifty feature, shadow buckets" so you get the gist of the article in the first two sentences